University-advisor-web/Controllers/MapController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using University_advisor_web.Models;
+
+// For more information on enabling MVC for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace University_advisor_web.Controllers
+{
+    public class MapController : Controller
+    {
+        // GET: /<controller>/
+        [HttpGet]
+        public IActionResult Index()
+        {
+            var model = new MapModel();
+            model.Address = "Vilnius";
+            model.Range = 500;
```

Comment:
As I understand these values are default and shown to user when he opens the page?

Reply:
Yes, I think in the future we should let user to input his home address and use it as the default value. Hardcoded value is temporary solution